snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -141,11 +141,7 @@ open class Snapshot: NSObject {
                 print("Couldn't find simulator home location. Please, check SIMULATOR_HOST_HOME env variable.")
                 return nil
             }
-            guard let homeDirUrl = URL(string: simulatorHostHome) else {
-                print("Can't prepare environment. Simulator home location is inaccessible. Does \(simulatorHostHome) exist?")
```

Comment:
I've testing changing the template on my project, haven't tried integrating from scratch again